cmd/pod-restarts-check/pod-restarts-check-clusterscope.yaml

```diff
@@ -0,0 +1,75 @@
+---
+# Source: kuberhealthy/templates/khcheck-pod-restarts.yaml
+apiVersion: comcast.github.io/v1
+kind: KuberhealthyCheck
+metadata:
+  name: pod-restarts
+  namespace: kuberhealthy
+spec:
+  runInterval: 5m
+  timeout: 10m
+  podSpec:
+    securityContext:
+      runAsUser: 999
+      fsGroup: 999
+    containers:
+      - env:
+          - name: MAX_FAILURES_ALLOWED
+            value: "10"
+        image: kuberhealthy/pod-restarts-check:v2.3.3
```

Comment:
I'm not sure if I need to do anything with this tag?  As this needs to points to a version that hasn't been released yet so maybe the release pipeline updates this?

Reply:
This one is actually manual... This is basically documentation.  The versions in the helm chart (and the flat files in the directory with the helm chart) get updated after we change the `version.yaml` of the chart, but this one does not.